<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeonchykCalculator</class>
 <widget class="QMainWindow" name="LeonchykCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>340</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="sevenButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border:1px solid grey;
}
QPushButton:pressed {
	background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eightButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border:1px solid grey;
}
QPushButton:pressed {
	background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nineButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border:1px solid grey;
}
QPushButton:pressed {
	background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fourButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border:1px solid grey;
}
QPushButton:pressed {
	background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fiveButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border:1px solid grey;
}
QPushButton:pressed {
	background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sixButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border:1px solid grey;
}
QPushButton:pressed {
	background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="oneButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border:1px solid grey;
}
QPushButton:pressed {
	background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="twoButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border:1px solid grey;
}
QPushButton:pressed {
	background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="threeButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border:1px solid grey;
}
QPushButton:pressed {
	background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zeroButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border:1px solid grey;
}
QPushButton:pressed {
	background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="equalButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:rgb(255,151,57);
	color: white;
	border:1px solid grey;
}
QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:rgb(215,215,215);
	border:1px solid grey;
}
QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minusButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:rgb(255,151,57);
	color: white;
	border:1px solid grey;
}
QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plusButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:rgb(255,151,57);
	color: white;
	border:1px solid grey;
}
QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="divideButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:rgb(255,151,57);
	color: white;
	border:1px solid grey;
}
QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sqrtButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:rgb(215,215,215);
	border:1px solid grey;
}
QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>Sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multiplyButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>270</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:rgb(255,151,57);
	color: white;
	border:1px solid grey;
}
QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dotButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:rgb(215,215,215);
	border:1px solid grey;
}
QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border:1px solid grey;
}
background-color:white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="negativeButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:rgb(215,215,215);
	border:1px solid grey;
}
QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="percentButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:rgb(215,215,215);
	border:1px solid grey;
}
QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SinButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:rgb(215,215,215);
	border:1px solid grey;
}
QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>Sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CosButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:rgb(215,215,215);
	border:1px solid grey;
}
QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>Cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TangButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:rgb(215,215,215);
	border:1px solid grey;
}
QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>Tang</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LogButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>340</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:rgb(215,215,215);
	border:1px solid grey;
}
QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x2Button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:rgb(255,151,57);
	color: white;
	border:1px solid grey;
}
QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>2x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x16Button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>370</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:rgb(255,151,57);
	color: white;
	border:1px solid grey;
}
QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>16x</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
